application/src/databaseengine.cpp

```diff
@@ -0,0 +1,135 @@
+#include "databaseengine.h"
+
+#include <QSql>
+#include <QSqlQuery>
+#include <QFile>
+#include <QDir>
+#include <QDebug>
+#include <QString>
+
+static QString STANDART_PATH = QDir::homePath();
```

Comment:
```suggestion
static QString STANDARD_PATH = QDir::homePath();
```
Только само название слишком общее, cache path или database path больше подходит по смыслу. Да и вообще в переменной homePath я не вижу смысла никакого

Reply:
Понял, исправлю